package.json

```diff
@@ -9,13 +9,14 @@
     "dist/*"
   ],
   "scripts": {
-    "clean": "npx rimraf node_modules .docz dist",
-    "build": "yarn clean && yarn install --ignore-optional && rollup -c",
-    "install-doc": "yarn clean && yarn install",
+    "install-default": "yarn install --ignore-optional",
+    "build": "rollup -c",
     "eslint": "eslint",
-    "docz:dev": "yarn install-doc && docz dev",
-    "docz:build": "yarn install-doc && docz build",
+    "docz:dev": "docz dev",
+    "docz:install": "yarn clean && yarn install",
```

Comment:
Could be interesting to use `yarn docz:clean` instead of `yarn clean`. 
`yarn docz:install` is used for `yarn docz:build` and `yarn docz:serve`, cached files in .docz/dist folders can be erroneous (specially when your're modifying documentation several times in a row)